pkgs/games/openrct2/engine.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchurl, cmake, pkgconfig
+, SDL2, SDL2_ttf, fontconfig, jansson, speexdsp, openssl, curl }:
```

Comment:
No, they are in sync with our `develop` branch. You are using older version of code.